### DISCUSSION TITLE
Deduction without clear reason

### DISCUSSION THREAD

**Discussion by chwjbn** [filecoin-project/lotus#7390] [Storage Provider] [answered]
### Lotus Version

```text
lotus version:1.10.0
```

### Describe the Bug

there are some coin deduction without clear reason,however,my lotus miner works well

my lotus miner:
https://filscan.io/address/miner?address=f01054527

### Logging Information

```text
FileShareLink：https://pan.baidu.com/s/18wlOCg3Uqb5wMx1G8ZPUNQ 
FileSharePassword：pmu9
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by LexLuthr** [comment, marked as answer]
@chwjbn Do you still need help with this or were you able to figure out why there was a deduction?

**Reply by chwjbn** [reply]
no need now,I have found out the reason
